BaseFilter/sections/specific.txt

```diff
@@ -346,6 +346,9 @@ $xmlhttprequest,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|s
 !
 ! NOTE: Specific rules
 !
+flightconnections.com##.free .display-box-2:not(#style_important)
```

Comment:
It seems that it's already hidden by:
```adblock
flightconnections.com#?#.display-box-2
```
from Base filter.

Reply:
Did not spot it, remove two rules that is not needed with base filter

Reply:
Now comment points to another line :')